jobs/garbage-collection.Jenkinsfile

```diff
@@ -0,0 +1,57 @@
+node {
+    checkout scm
+    // these are script global vars
+    pipeutils = load("utils.groovy")
+    pipecfg = pipeutils.load_pipecfg()
+}
+
+properties([
+    pipelineTriggers([]),
+    parameters([
+      choice(name: 'STREAM',
+             choices: pipeutils.get_streams_choices(pipecfg),
+             description: 'CoreOS stream to run GC')
+    ])
+])
+
+def build_description = "[${params.STREAM}] Garbage Collection"
+
+try {
+    currentBuild.description = "${build_description} Running"
+
+    credentials = [file(variable: "GCP_IMAGE_UPLOAD_CONFIG",
+                    credentialsId: "gcp-image-upload-config")]
+    // pick up the GCP project to use from the config
+    def gcp_project = shwrapCapture("jq -r .project_id \${GCP_KOLA_TESTS_CONFIG}")
+    def acl = pipecfg.s3.acl ?: 'public-read'
+
+    if (utils.credentialsExist(credentials)){
+        stage('Garbage Collection') {
+            pipeutils.shwrapWithAWSBuildUploadCredentials("""
+            cosa cloud-prune --policy ../gc-policy.yaml \
```

Comment:
Definitely sure this won't pickup the yaml file. Not sure how exactly we can utilise the file from the repo.

Reply:
Not sure if you are still blocked on this but looking at the pipeutils code the file should be in the working direcory because `piputils.load_pipecfg` already cloned the pipeline repo : 
https://github.com/coreos/fedora-coreos-pipeline/blob/main/utils.groovy#L25-L46

Reply:
This should be resolved now:)